Makefile

```
@@ -318,77 +326,54 @@ venv: ## Create Python virtual environment using uv
-		echo "Copying binary $(WHEEL_PLATFORM) $(WHEEL_BINARY)"; \
-		mkdir -p python-server/evalhub_server/binaries/; \
-		find python-server/evalhub_server/binaries/ -type f ! -name '.gitkeep' -delete; \
-		cp bin/$(WHEEL_BINARY)* python-server/evalhub_server/binaries/; \
-	else \
-		echo "Skipping copy (GITHUB_ACTIONS): binary provided by actions/download-artifact"; \
-	fi
-	@find python-server/evalhub_server/binaries/ -type f ! -name '.gitkeep' -exec chmod +x {} +
-	@echo "Building wheel for $(WHEEL_PLATFORM) with binary $(WHEEL_BINARY)..."
-	@rm -rf python-server/build/
-	@cp VERSION python-server/VERSION
+build-wheel: ## Build Python wheel: make build-wheel WHEEL_PLATFORM=manylinux_2_17_x86_64 CROSS_GOOS=linux CROSS_GOARCH=amd64
```

checkmake (0.3.2)

[warning] Target body for "build-wheel" exceeds allowed length of 5 lines (18).

(maxbodylength)